native/helloworld-rs/.gitignore

```
@@ -0,0 +1,3 @@
+/target/
```

This was aligned with what `cargo new` generates ([`cargo/src/cargo/ops/cargo_new.rs`](https://github.com/rust-lang/cargo/blob/d79fdf33ee52b250c0a81c3b5f00188a1e394620/src/cargo/ops/cargo_new.rs#L756)).

```suggestion
/target
```